js/src/edit-block/helpers/convertToString.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Internal dependencies
+ */
+import { TEXT_ARRAY_DELIMITER } from '../constants';
+
+/**
+ * Converts the setting to a string.
+ *
+ * @param {Array} settingValues The settings values.
+ * @return {string} The setting converted to a string.
+ */
+const convertToString = ( settingValues ) => {
+	if ( ! Array.isArray( settingValues ) ) {
+		return settingValues;
+	}
```

Comment:
If someone passed an object or other data type here, the function would return that instead of a string.

Reply:
Oh good idea!

Reply:
Committed in https://github.com/studiopress/genesis-custom-blocks/pull/41/commits/cc218836085b81b4d5a91fc720ca86e6536a6b66